java/HealthVault Mobile/HVSample/src/main/res/layout/add_medication.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent">
+
+	<!-- Name Labels/Text -->
+
+	<TextView
+		android:id="@+id/nameLabel"
+		android:layout_width="69dp"
+		android:layout_height="20dp"
+		android:layout_marginTop="20dp"
+		android:layout_marginLeft="20dp"
+		android:textColor="#767676"
+		android:textSize="15sp"
+		android:text="Name" />
```

Comment:
nit: Do we want to use strings.xml to organize all of these strings instead of using them inline? I imagine we don't have any need to localize, but might just be a better way to keep track of things, but not necessary